app.js

```diff
@@ -1,130 +1,178 @@
+        // h3's to names
+    productOneText = productOne.name;
+    productTwoText = productTwo.name;
+    productThreeText = productThree.name;
+
+    //img
+        // assign the DOM variable's src to the iamge propert of generated product because 
+    radioOneImage.src = productOne.image;
+    radioTwoImage.src = productTwo.image;
+    radioThreeImage.src = productThree.image;
+    
 };
 
-submitButton.addEventListener("click", () {
-  if (clicks > 0 && click < 25) {
-    displayResults()
-  }
-  let clicks = clicks += 1;
-  // getting ride of the items that previously appeared
-  getValidProducts(products, priorSet);
-  // this thius generates three random items and pushed them into an empty array
-  newSetFunction(validProducts);
-  // render set array on webpage
-  renderSetFunction(setArray);
+/* INVENTORY
+  I have: 
+          -connected the value of radio buttons to the id           of the prouct (this value means I can access the id of the product upon productSelectionCount
+          -Names of products stored in var
+          -src of images stored in variable
+          - a function that returns three randomly generated products
+          - a function that generates individual random products
+          - declared state variables
+          - a function that sets an intial state
+  */
 
 
 
+// 3. need an eventListener that will 
+    // call the display function()
+    // must have be able to increment the values of state variables: productSelectionCount ; total views ; total sets
+    // must discern between rendering new content OR ending survery
+        // conditions are if sets >= 25, survery ends and will DISPLAY RESULTS. If not, render new content and continue to increment.
 
+const form = document.querySelector('form')
+//this is creating a variable that is assigned to the value of the form and its contents
 
-  const userChoice = document.querySelector('input:checked')
-  click++ 
-});
 
+form.addEventListener("submit", (e) => {
+    e.preventDefault();
+    console.log('go');
+    
+    // conditions the circumsances of survey rendering
+    if (totalSets >= 25) {
+        displayResults()
+    }
 
-export const getValidProducts = (products, priorSet) => {
-  for (let i = 0; i < length.priorSet; i++) {
-      let i = priorSet[i];
-      let validProducts = products.splice(priorSet(i)); 
-  }
-  return validProducts;
-};
+    const formData = new FormData(form);
+    //this is taking in the form var, so the value of formData(var) is the data of the form itself upon event.
 
-export const setArrayFunction = (validProducts) => {
-    let setArray = []
-    let itemOne = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemOne.image);
-    let itemTwo = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemTwo.image);
-    let itemThree = Math.floor(Math.random() * validProducts.length);
-    setArray.push(itemThree.image)
+    const selectedProductId = formData.get('selected');
```

Comment:
when you're using a `FormData.get()` you're going to want to use the a string that matches the name name of the input in the HTML form.